src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -314,6 +317,47 @@ public void castIntFieldToDoubleWithAliasJdbcFormatGroupByTest() {
         }
     }
 
+    @Test
+    public void castKeywordFieldToDatetimeWithoutAliasJdbcFormatTest() {
+        JSONObject response = executeJdbcRequest("SELECT CAST(date_keyword AS DATETIME) FROM "
+                + TestsConstants.TEST_INDEX_DATE);
+
+        String date_type_cast = "{\"name\":\"cast_date_keyword\",\"type\":\"date\"}";
+        assertEquals(response.getJSONArray("schema").get(0).toString(), date_type_cast);
+        String[] expectedOutput = new String[] {"Tue Aug 19 07:09:13", "Wed Sep 25 02:04:13"};
+        String[] response_str = new String[2];
+
+        for (int i = 0; i < response.getJSONArray("datarows").length(); ++i) {
+            response_str[i] = response.getJSONArray("datarows").getJSONArray(i).getString(0);
+        }
+        Arrays.sort(response_str);
+        Arrays.sort(expectedOutput);
+        for (int i = 0; i < response_str.length; ++i) {
+            Assert.assertTrue(response_str[i].contains(expectedOutput[i]));
```

Comment:
If you control the data, why not put expected output into the set, then it's going to be easy to check that the set will contain every value? (here and everywhere)

Reply:
Good tip, thanks- will look into it